Source/ReorderController+SnapshotView.swift

```diff
@@ -28,7 +28,7 @@ extension ReorderController {
         guard let tableView = tableView, let superview = tableView.superview else { return }
         
         removeSnapshotView()
-        tableView.reloadData()
+        tableView.reloadRows(at: [indexPath], with: .fade)
```

Comment:
Thanks for putting this together. It looks like reloading with a `.fade` animation can cause some minor UI issues (you can see it on the 'effects' page of the demo app). Does the fix still work if you change the animation style to `.none`?

Reply:
It works with `none` too. I am going to update PR.